src/clib/pioc_sc.c

```diff
@@ -99,22 +99,36 @@ void compute_one_dim(int gdim, int ioprocs, int rank, PIO_Offset *start,
     irank = rank % ioprocs;
 
     /* Each IO task will have its share of the global dim. */
-    lcount = (long int)(gdim / ioprocs);
+    /* added ceil and float for Z5 */
```

Comment:
Remove commented code

Reply:
I changed these codes because I have to make the compute_one_dim distribute the count on each process evenly except the last count. So I should have a switch here to switch to your computation or z5 computation.